.github/workflows/mordor-tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: mordor tests
```

Comment:
we don't need to make the machine name public.
Both the filename and test can be called something like nvidia-<gpumodel> tests